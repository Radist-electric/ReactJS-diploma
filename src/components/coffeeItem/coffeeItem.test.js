import React from 'react';
import {Row} from 'reactstrap';
import {mount} from 'enzyme';
import CoffeeService from '../../services/coffeeService';

describe('Testing <CoffeeItem/>', () => {
    const service = new CoffeeService();
    const list = mount(<Row>
        {/* getData={service.getAllCoffee} */}
        onCoffeeLoaded={service.getAllCoffee}
        </Row>);
    // const item = shallow(<CoffeeItem/>);
    describe('Testing snap & state', () => {
        it('CoffeeItem has rendered correctly', () => {
            expect(list).toMatchSnapshot();
        });
    });
});