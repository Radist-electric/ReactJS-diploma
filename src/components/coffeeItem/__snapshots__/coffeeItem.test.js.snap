// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <CoffeeItem/> Testing snap & state CoffeeItem has rendered correctly 1`] = `
ReactWrapper {
  Symbol(enzyme.__unrendered__): <Row
    tag="div"
  >
    onCoffeeLoaded=
    [Function]
  </Row>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        "onCoffeeLoaded=",
        [Function],
      ],
      "tag": "div",
    },
    "ref": null,
    "rendered": Object {
      "instance": <div
        class="row"
      >
        onCoffeeLoaded=
      </div>,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "onCoffeeLoaded=",
          [Function],
        ],
        "className": "row",
      },
      "ref": null,
      "rendered": Array [
        "onCoffeeLoaded=",
      ],
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          "onCoffeeLoaded=",
          [Function],
        ],
        "tag": "div",
      },
      "ref": null,
      "rendered": Object {
        "instance": <div
          class="row"
        >
          onCoffeeLoaded=
        </div>,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "onCoffeeLoaded=",
            [Function],
          ],
          "className": "row",
        },
        "ref": null,
        "rendered": Array [
          "onCoffeeLoaded=",
        ],
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
